docs/README.md

```diff
@@ -55,6 +55,15 @@ All blocks exported by `store-link` share the same props:
 | `label`   | `string` | Link text                                                                                                                               | `undefined`   |
 | `href`    | `string` | Link URL                                                                                                                                | `'#'`         |
 | `target`  | `string` | This prop works the same way as the target of the [anchor element of html](https://developer.mozilla.org/pt-BR/docs/Web/HTML/Element/a) | `undefined`   |
+| `displayMode` | `DisplayMode` | How the link should be displayed | `'anchor'` |
+| `variant` | `Variant` | The variant to be used if `displayMode` is `button'` | `'primary'` |
```

Comment:
It might be a good idea to add a link to the Styleguide documentation for the Button since the values for `variant` can be found there